jpo-ode-core/src/main/java/us/dot/its/jpo/ode/model/OdeMapPayload.java

```diff
@@ -1,32 +1,32 @@
 package us.dot.its.jpo.ode.model;
 
```

Comment:
Updating classes that inherit OdeMsgPayload to now explicitly provide the OdeObject class as the type. This used to be the default, but had to change to allow for using ASN POJOs as part of the payload. Other check style changes were also applied.